mapdata/WarcraftLegacies/AbilityData/Skin/1162424385.json

```
@@ -12,7 +12,7 @@
     {
       "Id": 1718840929,
       "Type": 3,
-      "Value": "(Yellow - Falstad)"
+      "Value": "(Yellow - Falstad, Warsong - Varok)"
```

```suggestion
      "Value": "(Falstad, Varok)"
```
Pretty trivial but I think we should just stop putting faction names and colors in ability names when we're already putting the unit in. It's duplicate info.
(Good to add Varok in here though for sure)

---

Yup makes sense